README.md

```diff
@@ -37,6 +37,60 @@ docker cp -L ${container_id}:/bin/cpu_air_prover .
 docker cp -L ${container_id}:/bin/cpu_air_verifier .
 ```
 
+## Creating and verifying a proof of a Cairo program
+
+
+Navigate to the example test directory (`e2e_test/Cairo`):
+
+```bash
+cd e2e_test/Cairo
+```
+
+Install `cairo-vm/cairo1-run` (see further instructions in the
+[cairo-vm repository](https://github.com/lambdaclass/cairo-vm)):
+
+```bash
+git clone https://github.com/lambdaclass/cairo-vm.git
+cd cairo-vm/cairo1-run
+make deps
+```
+
+Compile and run the program to generate the prover input files:
+
+```bash
+cargo run ../../fibonacci.cairo \
+    --layout=small \
+    --air_public_input=fibonacci_public_input.json \
+    --air_private_input=fibonacci_private_input.json \
+    --trace_file=fibonacci_trace.json \
+    --memory_file=fibonacci_memory.json \
+    --proof_mode
+```
+
+Run the prover:
+```bash
+cpu_air_prover \
+    --out_file=fibonacci_proof.json \
+    --private_input_file=fibonacci_private_input.json \
+    --public_input_file=fibonacci_public_input.json \
+    --prover_config_file=../../cpu_air_prover_config.json \
```

Comment:
is '../..' a mistake?

Reply:
nope, you cd to e2e_test/cairo-vm/cairo1-run in the previous step and the configs are in e2e_test